link-area-delegation-explainer.md

```diff
@@ -0,0 +1,141 @@
+This is indeed a concern, but it's debatable whether enabling this feature in HTML makes a difference for this concern vs. the current approaches.
+The contrary might be true, where giving this information to the UA might give it a chance to intervene when the clickable nested target is too small.
+
+## Link Area Delegation
+The proposed mental model of this problem is "the card is delegating its interactive area to one of its link descendants".
+This can mean two things in terms of UI:
+1. The default click action for the card is the default click action of the link descendant
+2. The card's context menu behavior is augmented with the link descendant's context menu behavior (e.g. "Open in a new tab")
+3. Potentially, the UA can expand the hit-testing area for some of the descendants, to prioritize clicking them vs. clicking on the card.
+4. This feature has no effect on keyboard navigation.
+6. At first, this feature is not exposed to the accessibility tree, and should have no effect on assistive technologies. We can research in the future whether this card/link relationship can be expressed in these technologies in a useful way.
+
+### Link delegation attributes
+Expressing link area delegation in HTML can look something like this:
+```html
+<section class="card" linkarea>
+  <a href="/post?id=123" defaultlink>Post Title</a>
+  <img>
+  <button>Join</button>
+  <button>Share</button>
+</section>
+```
+
+In the above example, the `section` element has a `linkarea` attribute, which would delegate its click to the first descendant with a `defaultlink` attribute, in this case the first `a` element.
+
+We could have different similar alternatives to this with different ergonomic tradeoffs, e.g. using ID references:
+```html
+<section class="card" link="card123-title">
+  <a href="/post?id=123" id="card123-title">Post Title</a>
+  <img>
+  <button>Join</button>
+  <button>Share</button>
+</section>
+```
+
+Note that in this example, the link still has to be a descendant of the card. This is to guide web authors to an interactive structure where the links are in the appropriate place in the tree.
+
+## Alternatives considered
+
+### A new element type
+One of the alternatives that sound natural is a new element type:
+```html
+<section class="card">
+  <linkarea>
+    <a href="/post?id=123" defaultlink>Post Title</a>
+    <img>
+    <button>Join</button>
+    <button>Share</button>
+  </linkarea>
+</section>
+```
+
+This can feel compelling in a semantic sense, but becomes undesirable when we think of the flexibility of attributes.
+For example, a use case for link area delegation is making table rows clickable.
+
+```html
+<table>
+  <tr linkarea>
+     <td>1.</td>
+     <td><a href="/details?id=1" defaultlink>Title</a>
+  </tr>
+</table>
+```
```

Comment:
This use-case is a good reason to *not* limit this feature to link.

For example, the use-case might be to allow clicking the `checkbox` by clicking anywhere on a `tr`, which is not a link, but an input.

Perhaps the attribute should be `pointer-delegate`?

Reply:
Added a note